public/Day 22/style.css

```
@@ -105,9 +105,10 @@ body {
   background: linear-gradient(
     to bottom,
     transparent,
-    rgba(255,255,255,0.08),
+    rgba(0,0,0,0.08),
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)